<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginPage</class>
 <widget class="QMainWindow" name="LoginPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LoginPage</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="loginbutton">
    <property name="geometry">
     <rect>
      <x>411</x>
      <y>271</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="tabletTracking">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>LOGIN</string>
    </property>
   </widget>
   <widget class="QWidget" name="greenTop" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>211</height>
     </rect>
    </property>
    <widget class="QLabel" name="untlogo">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>0</y>
       <width>261</width>
       <height>211</height>
      </rect>
     </property>
     <property name="tabletTracking">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="whiteBack" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>210</y>
      <width>801</width>
      <height>361</height>
     </rect>
    </property>
    <widget class="QLabel" name="manageID">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>170</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Manage EUID and password</string>
     </property>
    </widget>
    <widget class="QLabel" name="stuhdnum">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>240</y>
       <width>191</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Student</string>
     </property>
    </widget>
    <widget class="QLabel" name="fachdnum">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>270</y>
       <width>231</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Faculty</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="showPassword">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>130</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Show Password</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
    </widget>
    <zorder>manageID</zorder>
    <zorder>fachdnum</zorder>
    <zorder>stuhdnum</zorder>
    <zorder>showPassword</zorder>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>251</x>
      <y>251</y>
      <width>137</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="IDlabel">
         <property name="text">
          <string>EUID</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="idtextbox_username">
         <property name="tabletTracking">
          <bool>true</bool>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="passtextbox">
         <property name="text">
          <string>PASSWORD</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="idtextbox_password">
         <property name="tabletTracking">
          <bool>true</bool>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <zorder>whiteBack</zorder>
   <zorder>loginbutton</zorder>
   <zorder>greenTop</zorder>
   <zorder>layoutWidget</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
